src/components/addBook.jsx

```diff
@@ -0,0 +1,5 @@
+const AddBook = () => (
+  <button type="submit">Add book</button>
```

Comment:
- [ ] Based on the requirements of this project, this button should include an event handler to add a book to the `books` array(with attributes title and author). Let's kindly adhere to the requirement of this project.